backend/server/repositories/bucket_repository.ts

```diff
@@ -1,11 +1,11 @@
-const Bucket = require('../models/bucket');
-const Promise = require('bluebird');
-const logger = require('../logger');
-const PageUtils = require('../utils/page_utils');
+const logger = require('tracer').colorConsole();
```

Comment:
Hmmmm, can we not find a way to configure this in one place then share it via require? Otherwise we cant configure what we want the logs to say and must stick with the default